ci/repo_cleanups/Jenkinsfile.garbage_collection

```diff
@@ -0,0 +1,26 @@
+#!groovy
+
+import org.jenkinsci.plugins.workflow.steps.FlowInterruptedException
+// http://stackoverflow.com/questions/37425064/how-to-use-environment-variables-in-a-groovy-function-using-a-jenkinsfile
+import groovy.transform.Field
+
+def checkout_and_merge() {
+    checkout scm
+    sh "git merge origin/master"
+}
```

Comment:
This function is not used (and isn't needed in the garbage collection job) so it should be removed.